superset/assets/javascripts/modules/utils.js

```diff
@@ -189,7 +189,7 @@ export function customizeToolTip(chart, xAxisFormatter, yAxisFormatters) {
   });
 }
 
-export function initJQueryAjaxCSRF() {
+export function initJQueryAjax() {
```

Comment:
there are actually multiple places need to handle 504 Gateway timeout problem for long query (but this ticket only fix for explore view). 

At the beginning I want to set a default jQuery ajax timeout limit from ajaxSetup, for all dashboard and explore view queries. It seems currently we only used this setup for CSRF, so it named as initJQueryAjaxCSRF. I am just thinking we might need a function to setup global, default jQuery ajax related settings, and this function is already hooked-up by many components.